pkg/testing/runner/supported.go

```diff
@@ -70,6 +72,26 @@ var (
 		},
 		Runner: DebianRunner{},
 	}
+	// RhelAMD64_8 - RedHat Enterprise Linux (amd64) 8
+	RhelAMD64_8 = SupportedOS{
+		OS: define.OS{
+			Type:    define.Linux,
+			Arch:    define.AMD64,
+			Distro:  Rhel,
+			Version: "8",
+		},
+		Runner: RhelRunner{},
+	}
+	// SlesAMD64_15 - SUSE Linux Enterprise Server (amd64) 15
+	SlesAMD64_15 = SupportedOS{
+		OS: define.OS{
+			Type:    define.Linux,
+			Arch:    define.AMD64,
+			Distro:  Sles,
+			Version: "15",
+		},
+		Runner: SlesRunner{},
+	}
```

Comment:
Same question, if SLES does not work for us, does it make sense to keep it in our tests?